mne/viz/tests/test_epochs.py

```diff
@@ -195,7 +184,32 @@ def test_plot_drop_log():
     plt.close('all')
 
 
-@requires_version('scipy', '0.12')
```

Comment:
was it intentional to remove this decorator from `test_plot_psd_epochs`?

Reply:
Yes I suggested this since we require a newer SciPy nowadays

Reply:
Yes, it was a suggestion of @larsoner because it appears to be outdated. The current release of scipy is much further.